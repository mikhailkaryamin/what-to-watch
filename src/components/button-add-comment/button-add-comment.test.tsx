import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {BrowserRouter as Router} from 'react-router-dom';

import ButtonAddComment from './button-add-comment';

describe(`ButtonAddComment render`, () => {
  const FILM_ID = 10;

  test(`should render buttonAddComment`, () => {
    const wrapper = renderer
      .create(
          <Router>
            <ButtonAddComment
              id={FILM_ID}
            />
          </Router>
      ).toJSON();

    expect(wrapper).toMatchSnapshot();
  });
});
