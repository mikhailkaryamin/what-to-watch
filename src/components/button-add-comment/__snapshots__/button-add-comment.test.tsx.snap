// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonAddComment render should render buttonAddComment 1`] = `
<a
  className="btn movie-card__button"
  href="/films/10/review"
  onClick={[Function]}
>
  Add review
</a>
`;
